cmake/Modules/Findcpp_redis.cmake

```diff
@@ -31,9 +31,16 @@ if (NOT cpp_redis_FOUND)
       TEST_COMMAND "" # remove test step
       )
   externalproject_get_property(cylix_cpp_redis source_dir binary_dir)
+
+  if(CMAKE_GENERATOR STREQUAL "Xcode")
+    set(cpp_redis_LIBRARY ${binary_dir}/lib/Debug/libcpp_redis.a)
```

Comment:
This should fail in case of `-DCMAKE_BUILD_TYPE=Release`. https://github.com/uic-evl/omegalib/blob/master/CMakeLists.txt#L87

Reply:
It should be 
```
set(cpp_redis_LIBRARY ${binary_dir}/lib/${CMAKE_BUILD_TYPE}/${CMAKE_STATIC_LIBRARY_PREFIX}cpp_redis${CMAKE_STATIC_LIBRARY_SUFFIX})
```

@lebdron what do you think?